spec/take_away_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'take_away' 
+describe TakeAway do 
+  let(:pasta) {double :meal, name: "pasta", price: 6}
+  let (:order) {double :order, meals: pasta}
+  let (:kenmenu) {double :menu}
+  let (:somenu){double :menu}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.